src/DynamoCoreWpf/Properties/Resources.en-US.resx

```diff
@@ -2565,6 +2569,32 @@ This package will be deleted after the next Dynamo restart.</value>
   <data name="PackageContextMenuUnmarkUnloadPackageTooltip" xml:space="preserve">
     <value>Remove the scheduled unload status</value>
   </data>
+  <data name="PackageViewContextMenuLoadText">
+    <value>Load</value>
+  </data>
+  <data name="PackageViewContextMenuLoadTooltip">
+    <value>Load this package into Dynamo. Other packages with the same name will be automatically deleted</value>
```

Comment:
I don't understand this. Loading a package will automatically delete other packages, isn't that bad?

Reply:
Does this mean if you try to load a built-in package, non-built-in packages with the same name will automatically be marked for deletion? And I think the built-in package will remain unloaded until you restart, correct?

Reply:
So in this case, the non-built-in package will have the `marked for deletion` state and the built-in will have the `scheduled for load` state?

Reply:
> So in this case, the non-built-in package will have the `marked for deletion` state and the built-in will have the `scheduled for load` state?

Correct.

In regards to the tooltip wording. I am not 100% happy with it. Not even sure we should make a distinction between Loading an automatic Unloaded package or a manually unloaded package.

Reply:
How about changing this to:
```
Load this package into Dynamo. Upon restart, other packages with the same name will be automatically deleted if they are custom packages or will be unloaded if they are built-in packages.
```

Reply:
Well, Load only applies to built in packages and conflicting packages can only be non-built in.
THan means that conflicting packages can only be deleted (never just unloaded)